projects/python/perception/facial_expression_recognition/image_based_facial_emotion_estimation/README.md

```diff
@@ -0,0 +1,96 @@
+# Image-based Facial Expression Recognition Demo
+
+This folder contains an implemented demo of image_based_facial_expression_recognition method implemented by [[1]](#1).
+The demo framework has three main features:
+- Image: recognizes facial expressions in images.
+- Video: recognizes facial expressions in videos in a frame-based approach.
+- Webcam: connects to a webcam and recognizes facial expressions of the closest face detected by a face detection algorithm.
+The demo utilizes OpenCV face detector Haar Cascade [[2]](https://ieeexplore.ieee.org/abstract/document/990517) for real-time face detection.
+
+#### Running demo
+The pretrained models on AffectNet Categorical dataset are provided by [[1]](#1) which can be found [here](https://github.com/siqueira-hc/Efficient-Facial-Feature-Learning-with-Wide-Ensemble-based-Convolutional-Neural-Networks/tree/master/model/ml/trained_models/esr_9).
+Please note that the pretrained weights cannot be used for commercial purposes!
+To recognize a facial expression in images, run the following command:
+```python
+python inference_demo.py image -i ./media/jackie.jpg -pre ./pretrained -d -s 2
```

Comment:
since these are the instructions on how to run the demo, I think we should include the image in the repo

Reply:
Preferably the image should be uploaded on the FTP and downloaded via the download method of the learner as "test_data".

Reply:
I uploaded a sample image and a sample video to the Owncloud which needs to be moved to the FTP server. In case the user does not provide an image/video path, it automatically downloads the sample image/video from the FTP server. 

The folder "demo" on Owncloud: 
`OpenDR/FTP Server Material/perception/facial_emotion_estimation/demo`

should move to FTP server: 
`perception/facial_emotion_estimation/`